src/main/scala/edu/ie3/simona/io/runtime/RuntimeEventLogSink.scala

```diff
@@ -26,6 +27,7 @@ import java.time.ZonedDateTime
 final case class RuntimeEventLogSink(
     simulationStartDate: ZonedDateTime,
     log: Logger,
+    private var last: Long = 0L,
```

Comment:
Using `var` is usually not what we want, but rather immutable values. We should look into this and try to find such a solution.